src/nanoemoji/config.py

```diff
@@ -200,6 +222,18 @@ def validate(self):
         if self.clipbox_quantization is not None and self.clipbox_quantization < 1:
             raise ValueError("If set, 'clipbox_quantization' must be 1 or positive")
 
+        # sanity check
+        assert self.has_svgs or self.has_bitmaps
+        # currently we only support picosvg-based vector+bitmap hybrid formats,
+        # no untouchedsvg+bitmap
+        assert not (self.has_bitmaps and self.has_untouchedsvgs)
```

Comment:
I dislike this one, I don't see a benefit to this failing the day I decide I do want an untouchedsvg_sbix_pancake font?

Reply:
yeah.. the reason being the code expects a .png to be found next to the input .svg, which is sort of ok for picosvg format for which we create build subfolders, not very much for untouchedsvg for which we use the original input svg file. OK, you convinced to change write_font to accept explicit .png inputs as well.